### ISSUE TITLE
💡  Add alignment controls for the Counter widget

### ISSUE THREAD

**Issue by JohnPixle** [elementor/elementor#9068]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
 Currently, the counter widget comes with centered content by default (number and title, however, many times it makes sense to have them left or right-aligned, depending the layout.

![Screenshot 2019-09-13 at 12 35 35](https://user-images.githubusercontent.com/4193672/64853161-8419e600-d623-11e9-877c-16c3c6ef1807.png)

**Describe the solution you'd like**
I think this alignment flexibility can be added painlessly with simply adding alignment controls in the Content tab.

In this quick mockup, you'll see the two alignment controls added.

![Screenshot 2019-09-13 at 12 46 07](https://user-images.githubusercontent.com/4193672/64853713-aa8c5100-d624-11e9-8e3c-4348c9132341.png)

✌🏻 You could also fit them under the styles tab.

**Describe alternatives you've considered**
Can be forced with css, no problem, but a native alignment control would work better.

**Additional context**
Thanks for considering!

**Comment by saranghills** [comment]
Yes, this is to be added please.

**Comment by gregzed** [comment]
Same please

**Comment by NilsDannemann** [comment]
Yes please.

**Comment by NilsDannemann** [comment]
Just as a quick fix for anyone who needs a solution to this right now - just add the following to the custom css in the settings for the counter widget:

```
selector .elementor-countdown-wrapper {
    text-align: left;
}
```

![image](https://user-images.githubusercontent.com/1572663/99906327-cb474080-2cd6-11eb-87b7-800909b9140d.png)

**Comment by maayannaveh** [comment]
Yes please!

**Comment by ianrandmckenzie** [comment]
I couldn't get @NilsDannemann 's code to work, but this seemed to work for me (but probably only works if you aren't using a prefix):
```css
selector .elementor-counter-number-prefix {
    flex-grow: 0;
}
```

**Comment by zorrobiwan** [comment]
And if you want to right align
```
selector .elementor-counter-number-suffix {
    flex-grow: 0;
}
```

Still a trick. real solution should be great

**Comment by AimeeHapp** [comment]
none of the above work for me. Please add an alignment option like you do on buttons and many other elements.

**Comment by hanna-III** [comment]
+1
would be very helpful

**Comment by fourstripes** [comment]
> //Number selector .elementor-counter-number-prefix { flex-grow: 0;
> 
> }
> 
> //Text
> 
> selector .elementor-counter-title{ text-align: left; }

Thank you! Legend. Very helpful.

**Comment by nicholaszein** [comment]
Hello everyone!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor v3.21.0! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
